exchanges/bithumb/bithumb_types.go

```diff
@@ -330,3 +324,10 @@ type StatusAll struct {
 	} `json:"data"`
 	Message string `json:"message"`
 }
+
+type OrderbookLevel struct {
```

Comment:
My mistake; Sorry.
None of `OrderbookLevel` or `OrderbookLevels`, or the functions on them, need to be public.
```suggestion
type orderbookLevel struct {
```

Reply:
Happy to keep all these exported as suggested by @gloriousCode